website/templates/public/pages/help/files.mako

```diff
@@ -5,8 +5,17 @@
         groups like datasets or background research.</p>
     <p>Each file is given a unique URL, meaning that it can be cited or linked to individually.</p>
     <p>To upload a file, click on "Files" in the grey navigation bar under the Project/Component's title. Here you
-        can drag the file from your desktop on to the screen to upload, or click on the upload button that appears after you've selected a storage add-on in the files pane. To download a file from a project, select the file and then click the Download button that appears in the toolbar; likewise, to delete a file, 
-        select the file and then click the Delete button. You can create folders to organize files by selecting the storage add-on in which you'd like to create a folder and clicking the Create Folder button in the toolbar.</p>
+        can drag the file from your desktop on to the screen to upload, or click on the upload button that appears 
+        after you've selected a storage add-on in the files pane. OSF Storage, the default storage add-on for every project, 
+        supports version control. This means that if you make changes to a file on your computer and save them&mdash;without 
```

Comment:
>  This means that if you make changes to a file on your computer and save them—without changing the name—you can drag and drop the newly updated file onto the OSF project where you are storing it and a new version will automatically be created

This sentence reads awkwardly; it can be simplified. Consider revising.